integ_test/test_scrapers.py

```diff
@@ -3,11 +3,10 @@
 import pytest
 
 import utils.dynamo
-from scrapers.hungary_hu import scrape_hungary_hu
-from scrapers.moldova_ro import scrape_moldova_ro
-from scrapers.poland import scrape_poland_en, scrape_poland_pl
-from scrapers.poland_en import scrape_poland_en
-from scrapers.poland_pl import scrape_poland_pl
+from scrapers.hungary_hu import HungaryScraper
+from scrapers.moldova_ro import MoldovaScraper
+from scrapers.poland import PolandScraper
+from scrapers.poland_en import PolandENScraper
```

Comment:
Should be good to delete this import?

Reply:
Oh yes, however I think it might be a good idea for a separate issue as to not polute this PR anymore?
I'll create an issue to remove the non-functional scrapers